app.rb

```diff
@@ -1,19 +1,81 @@
 require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
 apartments = data[:apartments]
 tenants = data[:tenants]
+ruby_apartments = []
+ruby_tenants = []
+
+ruby_apartments = apartments.each do |id, address, rent, size|
+   ruby_apartments.push(Apartment.new(id, address, rent, size))
+end
+
+ruby_tenants = tenants.each do |id, name, age, apartment_id|
+   ruby_tenants.push(Tenant.new(id, name, age, apartment_id))
+end
+
+puts "Welcome to MyAppartmentManager.com"
+puts "1) To View All Apartments"
+puts "2) To View All Tenants"
+puts "3) To Exit"
+
+selection = gets.chomp.to_i
+
+if selection == 1
+  puts ruby_apartments
```

Comment:
Suggest iterating over the apartments using `.each` and `puts`ing a nicer string about each apartment:

```ruby
ruby_apartments.each do |apt|
  puts "#{apt.address} costs #{apt.monthly_rent} per month and has #{apt.num_beds} beds."
end